go/vt/binlog/tables_filter_test.go

```diff
@@ -60,9 +61,7 @@ func TestTablesFilterPass(t *testing.T) {
 	})
 	_ = f(eventToken, statements)
 	want := `statement: <6, "set1"> statement: <7, "dml1 /* _stream included1 (id ) (500 ); */"> statement: <7, "dml2 /* _stream included2 (id ) (500 ); */"> position: "MariaDB/0-41983-1" `
-	if want != got {
-		t.Errorf("want\n%s, got\n%s", want, got)
-	}
+	assert.Equal(t, want, got, "binlogConnStreamer.tablesFilterFunc(): got:\n%+v\nwant:\n%+v", got, want)
```

Comment:
We should remove the `got:\n%+v\nwant:\n%+v` output, since `testify` prints the same kind of output anyway.